pydantic/errors.py

```diff
@@ -438,6 +440,21 @@ class DateError(PydanticValueError):
     msg_template = 'invalid date format'
 
 
+class _DateValueError(PydanticValueError):
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
You probably don't need this anymore and can make `DateNotInThePastError` and `DateNotInTheFutureError` inherit from `PydanticValueError`